app/forms.py

```diff
@@ -254,7 +262,6 @@ class NOIRegisterForm(RegisterForm):
     password = PasswordField(
         lazy_gettext('Password'), validators=[password_required,
                                               password_length])
-    password_confirm = None
```

Comment:
Removed this because `ConfirmRegisterForm` doesn't actually have a password confirmation field. See https://github.com/mattupstate/flask-security/issues/54#issuecomment-11938598 for more details.